EIPS/eip-2982.md

```diff
@@ -0,0 +1,197 @@
+
+## Specification
+
+Phase 0 is designed to require _no breaking consensus changes_ to existing Ethereum mainnet. Instead, this is an extension of Ethereum mainnet to bootstrap the new PoS consensus.
+
+Phase 0 specifications are maintained in a [repository](https://github.com/ethereum/eth2.0-specs) independent of this EIP. `SPEC_RELEASE_VERSION` release (TBD) of the specs at `SPEC_RELEASE_COMMIT` are considered the canonical Phase 0 specs for this EIP.
+
+This EIP provides a high level view on the Phase 0 mechanisms, especially those that are relevant to Ethereum mainnet (e.g. the deposit contract) and users (e.g. validator mechanics and eth2 issuance). The extended and low level details remain in the [specs repository](https://github.com/ethereum/eth2.0-specs).
+
+### Parameters
+
+| Parameter | Value |
+| - | - |
+| `SPEC_RELEASE_VERSION` | TBD |
+| `SPEC_RELEASE_COMMIT` | TBD |
+| `DEPOSIT_CONTRACT_ADDRESS` | TBD |
+| `MIN_GENESIS_TIME` | TBD |
+| `BASE_REWARD_FACTOR` | `2**6` (64) |
+| `INACTIVITY_PENALTY_QUOTIENT` | `2**26` (67,108,864) |
+| `PROPORTIONAL_SLASHING_MULTIPLIER` | `1` |
+| `MIN_SLASHING_PENALTY_QUOTIENT` | `2**5` (128) | 
+
+_Note:_ Eth2 has many more [Phase 0 configuration parameters](https://github.com/ethereum/eth2.0-specs/blob/dev/configs/mainnet/phase0.yaml) but the majority are left out of this EIP for brevity.
+
+### Validator deposit contract
+
+In Phase 0, eth2 uses a contract deployed on Ethereum mainnet -- the [Deposit Contract](https://github.com/ethereum/eth2.0-specs/blob/dev/specs/phase0/deposit-contract.md) -- at `DEPOSIT_CONTRACT_ADDRESS` to onboard validators into the PoS consensus of the beacon chain.
+
+To participate in the PoS consensus, users submit validator deposits to the deposit contract. The beacon chain comes to consensus on the state of this contract and processes new validator deposits. This uni-directional deposit mechanism is the only technical link between the two components of the system (Ethereum mainnet and beacon chain) in Phase 0.
+
+More on this mechanism:
+* [Deposit Contract](https://github.com/ethereum/eth2.0-specs/blob/dev/specs/phase0/deposit-contract.md)
+* [Beacon Chain -- `Deposit` processing](https://github.com/ethereum/eth2.0-specs/blob/dev/specs/phase0/beacon-chain.md#deposits)
+* [Validator -- Becoming a validator](https://github.com/ethereum/eth2.0-specs/blob/dev/specs/phase0/validator.md#becoming-a-validator)
+
+### Beacon chain and validator mechanics
+
+Users who choose to participate in eth2 consensus deposit ETH collateral into the [deposit contract](https://github.com/ethereum/eth2.0-specs/blob/dev/specs/phase0/deposit-contract.md) in order to be inducted into the beacon chain validator set. From there, these validators are responsible for constructing the **beacon chain** (note that these consensus participants in PoS are akin to miners in PoW).
```

Comment:
Why is it possible to deposit funds into a contract for a chain that doesn't yet do anything?  Or am I misunderstanding?

Reply:
Phase 0 is the bootstrapping phase of the PoS consensus mechanism intended to drive Ethereum's consensus. Ethereum handles billions of dollars worth of activity, so this upgrade is designed in a relatively conservative path to first bootstrap the new consensus and once it is stable in production, replace Ethereum's PoW for the PoS of the beacon chain.

The deposit contract serves to allow validators to enter into this new consensus mechanism and participate with real economic value to run the core of the mechanism -- building and maintaining it's own consensus via Casper FFG/LMD-GHOST. 

So to answer your question "Why is it possible to deposit funds into a contract for a chain that doesn't yet do anything?" --  these early deposits serve to bootstrap the consensus and to execute the core algorithms in a production context. This "chain that doesn't yet do anything" actually does quite a bit in terms of consensus and mechanism design, and the core can be extended to come to consensus on user level activity (the activity we enjoy today on Ethereum mainnet).

Only after the core consensus algorithms are run successfully and stably in production will Ethereum's PoW consensus be swapped for that of the beacon chain.

The EIP does attempt to address and explain this via the following. If that is not sufficient, I'm open to advice on how to make this more clear.

> This EIP specifies Phase 0 of Serenity (eth2), a multi-phased upgrade to the consensus mechanism for Ethereum mainnet. In Phase 0, the existing PoW chain and mechanics are entirely unaffected, while a PoS chain -- the beacon chain -- is built in parallel to serve as the core of the upgraded consensus. In subsequent phases, the beacon chain is enhanced to support and secure the consensus of a number of parallel shard chains, ultimately incorporating current Ethereum mainnet as one of those shards.

> At the core of the beacon chain is a proof of stake consensus mechanism called Casper the Friendly Finality Gadget (FFG) and a fork-choice rule called Latest Message Driven Greedy Heaviest Observed Sub-Tree (LMD-GHOST). Phase 0 is centered primarily around the mechanics and incentives of validators executing these algorithms.

Reply:
Thanks Danny.  "Doesn't do anything" was harsh, my apologies.  What Phase 0 doesn't seem to do is let users spend or withdraw the ETH they have locked up, run contracts, or come to consensus on anything except the operation of the chain itself.  I'm not clear why valuable funds need to be locked up for that against the promise of future deliverables.  In fact, I fear this is exactly the sort of thing that could make ETH look like a security to the SEC and similar authorities - we could pass the infamous Howey test that we have flunked so far.
1. It is an investment of money.
2. There is an expectation of profits from the investment.
3. The investment of money is in a common enterprise.
4. Any profit comes from the efforts of a promoter or third party.

Reply:
@gcolvin I don't think this PR is the proper venue for bringing up these issues about Eth 2 Phase 0 itself, especially if it is now an Informational EIP. We have the ethresear.ch forum for Eth2 discussion and I assume there are threads there that address your commentary.

Reply:
@Souptacular Unfortunately, I find myself in a place similar to that Yoichi was in - this EIP describes what I fear could be counted as being or enabling securities fraud - which is not possible so long as ETH is not considered a security.  And nobody can tell me whether it is or might become such.  Making this EIP merely informational might help, but the information is easily available elsewhere.  So why take on risk here?  As the old maps said about uncharted oceans, "Here be dragons."

Reply:
I can't tell you how to feel about this, but I can say how I'm approaching this and what I feel like your options are.

#1: If what is described in this EIP is the future direction of Ethereum the venue for the document shouldn't make much of a difference in enabling securities fraud. As you've said, the info is "easily available elsewhere."
#2: Most important: As EIP editors, we are violating EIP-1 if we pass judgement and prevent the inclusion of an EIP based on personal judgements. EIP-1 states, "The editors don't pass judgment on EIPs. We merely do the administrative & editorial part." The EIP editors are not securities lawyers and preventing an EIP, especially an Informational EIP, from being published is akin to censorship.

You completely are within your right to abstain from being an editor for an EIP you disagree with or feel is risky. After years of discussion, that is publicly available, about Eth 2.0 research and development I think it's not acceptable to use this PR as the venue for your concerns.

Reply:
@Souptacular @MicahZoltu  Taking this to the Editor's chat room (Skype for me.)

Reply:
To be clear, **I am not accusing anyone of fraud**.  My concern is that (in my understanding) the law can be retroactive.  If an authority determines that something is a security, then it can turn out that you were selling unregistered securities all along.